caldp/20220214/CALDP_20220214_COS_macos_py38_final.yml

```diff
@@ -0,0 +1,104 @@
+# delivery_name: CALDP_20220214_COS_final
+# creation_time: Mon Feb 14 15:00:00 2022
+# conda_version: 4.10.3
+# condabuild_version: 3.18.11
+channels:
+  - https://ssb.stsci.edu/astroconda-staging
+  - https://ssb.stsci.edu/astroconda
+  - defaults
```

Comment:
Why is conda-forge missing from the finalized macos release? Also, the release is using the staging area's hstcal package in both cases, rather than conda-forge's based on the channel order. Is this intended?

(I'm only mentioning it because there's no point to include conda-forge if it isn't being used)

Reply:
This is a "franken-build" in that the YML from caldp_20211129 was used as the basis for the build.  Please note this YML was generated with the current "astroconda" build system and has `hstcal=2.7.2.dev0+gd395b8b8=0` .  A conda environment was created using caldp_20211129, calcos was updated within this environment to v3.4.0, and then a new output YML was created from this updated environment.  Since caldp_20211129 had already been vetted by the INS teams as a good build, it was felt by creating an updated build environment in this manner, this environment would only contain software that would affect the COS data.  As such, the regression testing for the INS teams should (!) go more quickly So....conda-forge is really not in play (yet).

The reason for this non-standard process is the CALCOS build is needed as soon as yesterday.   The MVMALPHA build sequence ended up taking much longer than anticipated, and the CALCOS build had to by-pass the MVMALPHA build to get out the door in order to have time for instrument team testing.  COS LP6 data is expected in early March and operations is waiting for the CALCOS update.

Reply:
We haven't pushed the changes necessary for conda-forge in the build yet, so it shouldn't be there yet. _Hopefully_ will have that in play for MVMAlpha.